content/docs/1.7.0/references/reference-setup-performance-scalability-and-sizing-guidelines.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Reference Setup, Performance, Scalability, and Sizing Guidelines
+weight: 2
+---
+
+You can find the full details at https://github.com/longhorn/longhorn/tree/v{{< current-version >}}/scalability/reference-setup-performance-scalability-and-sizing-guidelines
```

Comment:
```suggestion
You can find the detailed guidelines in the [Longhorn repository](https://github.com/longhorn/longhorn/tree/v{{< current-version >}}/scalability/reference-setup-performance-scalability-and-sizing-guidelines).

Longhorn can be deployed in environments with varying specifications, so testing for all viable setups is not possible. Because of this, the guidelines are based on typical setups for on-premises and public cloud deployments.
```

Reply:
I added the second paragraph to set user expectations about the scope of the linked content.

Reply:
Thank you. I fixed the first paragraph.

I think we don't need the 2nd paragraph because the title of the page already mentions the context for the link which is `Reference Setup, Performance, Scalability, and Sizing Guidelines` . Furthermore, `Longhorn can be deployed in environments with varying specifications, so testing for all viable setups is not possible. Because of this, the guidelines are based on typical setups for on-premises and public cloud deployments.` seems to be specific testing details not the summary of the goals of the report. WDYT?